### ISSUE TITLE
Mix live streamed audio 

### ISSUE THREAD

**Issue by mgw-sbic** [alemangui/pizzicato#134]
Hi Pizzicato team,

We'd like to find a way to mix real time live audio feeds that are arriving via streaming. Is there a way to associate mixer inputs with live streams somehow?

**Comment by sorcerykid** [comment]
I am wondering this too. I setup a SHOUTcast server and tried to play the live audio feed via Pizzicato, and it doesn't seem to work no which method I try. Given how old this issue is, it looks like the developers have no plans to support streaming audio.

**Attempt 1**
```
<script>
var audio = new Pizzicato.Sound("http://www.example.com:8000/example?type=http&amp;nocache=4", function() {
        alert("loaded sound");
});
</script>
```

**Attempt 2**
```
<audio id="audio_source">
        <source src="http://example.com:8000/example?type=http&amp;nocache=4" type='audio/mpeg'>
</audio>
<script>
var audio = document.getElementById("audio_source");
var group = new Pizzicato.Group([audio]);
</script>
```

**Comment by mgw-sbic** [comment]
Yes they said it would take a major revision. It would be so awesome though!

**Comment by sorcerykid** [comment]
I successfully managed to work around the issue by creating my own AudioContext and applying effects manually, therefore obviating the need for Pizzicato or any third-party JS library completely.

Here's some boilerplate code I developed that has been tested with a live SHOUTcast server. It chains several different audio nodes including Convolver, StereoPanner, BiQuadFilter, and Gain, and it provides dry/wet effects loop mixing for the Convolver.

https://gist.github.com/sorcerykid/a6cb75122b9522607821857c65313501

Note that the convolver requires an impulse response waveform which can be readily obtained from various sources online.
